app/channel.go

```diff
@@ -185,8 +185,11 @@ func (a *App) JoinDefaultChannels(c request.CTX, teamID string, user *model.User
 
 		a.invalidateCacheForChannelMembers(channel.Id)
 
+		options := a.Config().GetSanitizeOptions()
+		user.SanitizeProfile(options)
+
 		message := model.NewWebSocketEvent(model.WebsocketEventUserAdded, "", channel.Id, "", nil, "")
-		message.Add("user_id", user.Id)
```

Comment:
In order to be backwards compatible, keep the `user_id` also in the message. So the message will have:
`user_id`, `user` and `team_id`.

Reply:
Done

Thank you for your support @larkox